src/hotspot/share/opto/macro.cpp

```diff
@@ -2414,31 +2359,31 @@ void PhaseMacroExpand::expand_lock_node(LockNode *lock) {
                                   OptoRuntime::complete_monitor_locking_Java(), NULL, slow_path,
                                   obj, box, NULL);
 
-  extract_call_projections(call);
+  call->extract_projections(&_callprojs, false/*separate_io_proj*/, false/*do_asserts*/);
 
   // Slow path can only throw asynchronous exceptions, which are always
   // de-opted.  So the compiler thinks the slow-call can never throw an
   // exception.  If it DOES throw an exception we would need the debug
   // info removed first (since if it throws there is no monitor).
-  assert ( _ioproj_fallthrough == NULL && _ioproj_catchall == NULL &&
-           _memproj_catchall == NULL && _catchallcatchproj == NULL, "Unexpected projection from Lock");
+  assert ( _callprojs.fallthrough_ioproj == NULL && _callprojs.catchall_ioproj == NULL &&
```

Comment:
Whitespaces should be removed `assert ( _c` -> `assert(_c`

Reply:
fixed.